assets/javascripts/lib/discourse-markdown/bbcode-color.js.es6

```diff
@@ -21,7 +21,13 @@ function replaceFontBgColor(text) {
 }
 
 export function setup(helper) {
-  helper.whiteList(['font[color]']);
+  helper.whiteList({
+    custom(tag, name, value) {
+      if (tag === 'span' && name === 'style') {
+        return /^color:.*$/.exec(value);
```

Comment:
If you fix this @iamravenous, please, can you also fix the `background-color` whitelist too? (I'm sure you can combine both)